### PR TITLE
docs: typos fixed

### PR DESCRIPTION
**Pull request by annehubtype**
_Remember to tag the PR with **WIP** tag if it's not ready to be merged_.  
[Reference](https://blog.codeminer42.com/on-writing-a-great-pull-request-37c60ce6f31d/)

## Description

Fixed English typos in Botonic documentation throughout the site. Please check if everything's fine. Thanks.

## Testing

The pull request...

- [ ] has unit tests
- [ ] has integration tests
- [ ] doesn't need tests because... **[provide a description]**

### CODE REVIEW

**Comment by dpinol** [comment]
you have many conflicts. Did you start the branch from a previous PR?
let's review it together if you have doubts

File: doc-botonic/docs/components/components-carousel.md [hunk 1]
```diff
@@ -5,11 +5,13 @@ title: Carousel
 
 ## Purpose
 
-A carousel is a sliding gallery of information displayed to the user. It helps to provide more information than that which fits the screen.
+Carousels show a collection of clickable items one at a time where the user can scroll images or articles from one side to the other.
```
[the remark this answers is not in the document]
Comment (annehubtype) [reply doc-botonic/docs/components/components-carousel.md]: Corrected

File: doc-botonic/docs/concepts/14.humanhandoff.md [hunk 2]
```diff
@@ -145,6 +144,61 @@ export default class extends React.Component {
 }
 ```
 
-**Note:** For getting the agents with extra information, sorted by the number of cases that
-are attending in Hubtype, you can use `getAvailableAgents`.
+**Note:** To get extra information from the agents, sorted by the number of cases that are attending in Hubtype, you can use `getAvailableAgents`.
+
+## Human Handoff Template
+
+To set up human handoff, you can install Botonic by using the relevant template with a very simple use case where a user is transferred to a human agent through the chatbot. The conversation is then displayed as a new case on Hubtype Desk.
```
[the remark this answers is not in the document]
Comment (annehubtype) [reply doc-botonic/docs/concepts/14.humanhandoff.md]: Sentence changed.

File: doc-botonic/docs/concepts/14.humanhandoff.md [hunk 3]
```diff
@@ -145,6 +144,61 @@ export default class extends React.Component {
...
+        <Text>
+          Sorry, right now we can't serve you... Please contact us later!
+        </Text>
+      )
+    }
+  }
+}
+```
 
+**Note:** Remember that the full features for `getOpenQueues` and `humanHandOff` are only available when the bot is deployed!
```
[the remark this answers is not in the document]
Comment (annehubtype) [reply doc-botonic/docs/concepts/14.humanhandoff.md:204]: Removed from template

File: doc-botonic/docs/concepts/14.humanhandoff.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (annehubtype) [reply doc-botonic/docs/concepts/14.humanhandoff.md]: It's OK.

File: doc-botonic/docs/create-convapp.md [hunk 4]
```diff
@@ -5,187 +5,211 @@ title: Create a Conversational App from Scratch
...
    ```javascript
-   import ChosenPizza from "./actions/chosen-pizza";
-   import ChosenPasta from "./actions/chosen-pasta";
+   import ChosenPizza from './actions/chosen-pizza'
+   import ChosenPasta from './actions/chosen-pasta'
    ```
 
-7. Then in `routes.js`, you can add rules to make the user follow a logical path such as a **payload** to capture the answer once the user has clicked on the button `{ payload: "pizza", action: ChosenPizza }` or a **sub-flow** where you can select between two options via `childRoutes`.  
-   **Note:** When using `childRoutes`, at least the main action must have been previously passed. 
+1. Then in `src/routes.js`, you can add rules to make the user follow a logical path such as a **payload** to capture the answer once the user has clicked on the button `{ payload: "pizza", action: ChosenPizza }` or a **sub-flow** where you can select between two options via `childRoutes`.
```
[the remark this answers is not in the document]
Comment (annehubtype) [reply doc-botonic/docs/create-convapp.md]: Sentence split.

File: doc-botonic/docs/create-convapp.md [hunk 5]
```diff
@@ -5,187 +5,211 @@ title: Create a Conversational App from Scratch
...
-];
+]
 ```
 
 ## Add NLU
 
 1. In the bot directory, enter `npm install @botonic/plugin-nlu`.
 
-2. Create two types of intentions, i.e. two text files called `chitchat.txt` and `help.txt` in `src/nlu/utterances/en`. You must create at least **two files** to make the NLU plugin work.
+2. Create two types of `intents`, i.e. two text files called `chitchat.txt` and `help.txt` in `src/nlu/utterances/en` containing utterances related to an intention. You must create at least **two files** to make the NLU plugin work.
```
[the remark this answers is not in the document]
Comment (annehubtype) [reply doc-botonic/docs/create-convapp.md]: Intent used.

**Pull request merged**